pkg/operator/hostetcdendpointcontroller/members.go

```diff
@@ -163,3 +177,11 @@ func (h *healthyEtcdMemberGetter) Endpoints() ([]string, error) {
 
 	return endpoints, nil
 }
+
```

Comment:
question: would it be fair to add a TODO here to util this function for using it across the projects?

Reply:
agreed

Reply:
done